src/KOKKOS/sna_kokkos_impl.h

```diff
@@ -231,11 +232,22 @@ void SNAKokkos<DeviceType>::grow_rij(int newnatom, int newnmax)
   zlist = t_sna_2c_ll("sna:zlist",idxz_max,natom);
 
   //ulist = t_sna_3c("sna:ulist",natom,nmax,idxu_max);
-  ulist = t_sna_3c_ll("sna:ulist",idxu_max,natom,nmax);
+#ifdef KOKKOS_ENABLE_CUDA
+  if (std::is_same<DeviceType,Kokkos::Cuda>::value) {
+    // dummy allocation
+    ulist = t_sna_3c_ll("sna:ulist",1,1,1);
```

Comment:
@weinbe2 do we need this dummy allocation, or could it be removed?

Reply:
@stanmoore1 I don't see why it couldn't be removed, I was just irked by the idea of having an uninitialized View. That doesn't mean my way of addressing that was smart, though...

Reply:
OK I see your point, I'll think about it for a bit.

Reply:
Let's leave this for now.